### PR TITLE
Subnautica Stream

### PR DESCRIPTION
**Pull request by Darkeew**

### CODE REVIEW

File: src/prompts/streams/vedal/sub.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    neuro.chatPrompts,
+    vedal.chatPrompts,
+    chat.chatPrompts
+  ).filter(x => !chatPrompts.includes(x)) // removes duplicated chat prompts
+}
+
+export default {
+  name: 'Subnautica',
+  // CHANGE THIS
+  image: 'vedal/vedalSpaceProgram.png',
```
Comment (tioopental) [inline src/prompts/streams/vedal/sub.js:70]:
```suggestion
  image: 'vedal/neurovedalsubmarine.png',
```

File: src/prompts/streams/vedal/sub.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  prompts: vedal.regularPrompts
+}
+//  REPLACE THIS
+const chatPrompts = [
+  'chat spams :PANIC:',
+  'chat spams :VedalSpaceProgram:',
+  'chat spams :EZ: :Clap:',
+  'chat spams :peepoCheer:',
+  'chat spams :o7: :neuro7:'
+]
```
Comment (tioopental) [inline src/prompts/streams/vedal/sub.js:27]:
```suggestion
const chatPrompts = [
  'chat spams :PANIC:',
  'chat spams :EZ: :Clap:',
  'chat spams :o7: :neuro7:'
]
```

**Review by tioopental — APPROVED** [review]
image

File: src/prompts/streams/vedal/sub.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    'gets killed by\xa0fish',
+    'game crashed',
+    'runs out of\xa0oxygen',
+    'jumpscared',
+    'dies of starvation',
+    'dies of hydration',
+    'finds a leviathan',
+    'forgets where he is',
+    'stuck inside a cave',
+    'Neuro complains with Vedal',
```
Comment (cjmaxik) [inline src/prompts/streams/vedal/sub.js:48]: What does this mean?

**Pull request merged**